# Goal
string object_name
---
# Result
int32 SUCCESS = 0
int32 ABORTED = 1
int32 REJECTED = 2
int32 result
---
# Feedback message
geometry_msgs/Pose current_pose